connectors/src/lib/models.ts

```diff
@@ -106,10 +104,35 @@ Connector.init(
       type: DataTypes.STRING,
       allowNull: true,
     },
+    // must be set for all nango-based connectors
+    nangoConnectionId: {
```

Comment:
how about renaming nangoConnectionId => connectionId
and use that for github?

Reply:
Yeah a bit but should be fine and it's connectors so we have a margin of error :+1: